afidsvalidator/__init__.py

```
@@ -4,9 +4,20 @@
 
 from flask import Flask
 
+from config import *
+
 from afidsvalidator.views import validator
 from afidsvalidator.model import db
 
+# Grab environment
```

Can we raise an exception if FLASK_ENV isn't defined? Something like:
```python
if os.environ.get("FLASK_ENV") is None:
    # Raise an exception
```

---

Done